### ISSUE TITLE
Alexa media player incorrectly reporting media and bluetoth status 

### ISSUE THREAD

**Issue by cunninr2** [custom-components/alexa_media_player#392]
**Describe the bug**
In creating automations using alexa media player I have noticed some inconsistent data retrieval regarding the status of alexa devices which prevent me from completing an automation. This concerns the reporting of the bluetooth source list and the active source. 

**To Reproduce**
1. Asks any alexa device to connect to a paired bluetooth audio device and wait for a successful connection.
2. In home assistant, look at the state information for the alexa device. You will see that the source list for that alexa device is often not correct and you will also see that the source is not updated to reflect the bluetooth connection.
3. My use case here was to turn on a yamaha AVR and set it to the bluetooth input once the alexa device had confirmed a successful connection to a bluetooth audio receiver.
4. There are no logs regarding this issue as it is concerned only with the correct reporting of the alexa device metadata.

**Expected behavior**
I  expected the meta data to update in accordance with the actual status of the device.

**System details**
 - Home-assistant (version): 0.99.2
 - Hassio :Yes
 - alexa_media  (version from `const.py` or HA startup): v.2.2.1
 - alexapy (version from `pip show alexapy` or HA startup): very latest version

**Logs**
No log information forthcoming to indicate a problem.

**Comment by alandtse** [comment]
To confirm, this is for pairing sources (input) to Alexa and not sinks (output to another speaker), right?

This should already be happening but it's possible there's a bug. Would need some logs with alexapy showing the steps. Assuming websocket is connected, you'd see the actions in the logs.

Are media player changes displaying for you?

**Comment by alandtse** [comment]
Please try 2.3.1

**Comment by WilliamMolina** [comment]
@cunninr2, your problem is with source device or sink device?

I had a similar issue with a bluetooth speaker. I reviewed the code of media_player.py and I found that only A2DP_SOURCE devices are listed as sources, and my bluetooth speaker was profiled as A2DP_SINK, so I changed the code to list my A2DP_SINK devices and it worked.

I am going to test that change and create a pull request.

If you are still having trouble, feel free to contact me.
